selfdrive/car/hyundai/carstate.py

```diff
@@ -191,17 +197,16 @@ def update(self, cp, cp_cam):
     self.left_blinker_flash = cp.vl["CGW1"]['CF_Gway_TurnSigLh']
     self.right_blinker_on = cp.vl["CGW1"]['CF_Gway_TSigRHSw']
     self.right_blinker_flash = cp.vl["CGW1"]['CF_Gway_TurnSigRh']
-    self.steer_override = abs(cp.vl["MDPS11"]['CR_Mdps_DrvTq']) > 100.
+    self.steer_override = abs(cp.vl["MDPS12"]['CR_Mdps_StrColTq']) > 1.0
```

Comment:
StrColTq is a scaled value, so the threshold of touching the wheel also needed changing

Reply:
is CR_Mdps_StrColTq the same as CR_Mdps_DrvTq (except for scaling?)

Reply:
Identical
![screenshot-20181227183632-988x1166](https://user-images.githubusercontent.com/5079957/50470858-66114480-0a06-11e9-913a-548828b80a54.png)

Reply:
cool